### ISSUE TITLE
it is not showing any of the plannets

### ISSUE THREAD

**Issue by StoianAndrei** [calli-eve/eve-pi#6]
Good evening, may I ask if the website will be down or is just a bug? after log in the main tab with totals do not show any more the plannets that are due for renewal. 

Hope you fix it. Let us know if you need any help. Lovely site. Very usefull.

Thanks

**Comment by calli-eve** [comment]
Hey! Can you check that you don't have Alert Mode on? That filters out all planets that have no extractors that need restarting. This could be also due to refresh token going stale due to some time of inactivity and that we can't do much about because backend does not store tokens. You'll need to relogin the characters. 

If you could check these and provide some error logs from your browser console so we can look into it :)

**Comment by StoianAndrei** [comment]
![image](https://github.com/calli-eve/eve-pi/assets/47652458/29478e87-c6c7-4a77-9878-4ffa0f46d030)

**Comment by StoianAndrei** [comment]
Hey there so I did thought of if and also re-auth with all chars so that we have fresh new tokens. very wierd. I took at one point the docker version of it and it did work. But the website one was so convinient. Let me know if need me to do something else. Thanks again.

**Comment by calli-eve** [comment]
So everything worked after reauth?

**Comment by calli-eve** [comment]
Actually found the most probable issue. Refresh token expiration was not handled proprely and this made the whole thing crash if one token had expired. This was not a problem before, because each character loaded their planets separately and you just did not see any planets for the expired character. I hoisted the state loading to root a couple of weeks ago to get the totals calculations going and now the Promise.all(refresh) caused the whole thing to crash and burn if one token was expired.  Fixed in the latest commit: https://github.com/calli-eve/eve-pi/commit/793f71476bad969317c15de23feda5808f8c2cf8
